### PR TITLE
0.4.0 release

### PR DESCRIPTION
**Pull request by mzruya**

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -17,3 +17,9 @@ appraise 'rails-6' do
   gem 'activesupport', '~> 6.0'
   gem 'sqlite3', '~> 1.4'
 end
+
+appraise 'rails-edge' do
```
Comment (mzruya) [inline Appraisals:21]: @kellysutton, wdyt?

Comment (kellysutton) [reply Appraisals:21]: Let's do it

**Comment by mzruya** [comment]
@JackDanger 0.4.0 is available on https://rubygems.org/gems/ar-query-matchers/versions/0.4.0.

**Comment by JackDanger** [comment]
@mzruya 🙇🏻‍♂️

**Pull request merged**